### ISSUE TITLE
Java 9/10 whole number versions

### ISSUE THREAD

**Issue by zman0900** [jenv/jenv#204]
Adding the original releases of java 9 or 10 creates version aliases like "9" and "10".  But adding the later updates such as 9.0.4 and 10.0.1 creates version aliases like "9.0" and "10.0".  These later versions should also create aliases like "9" and "10" to simplify upgrades.

**Comment by eed3si9n** [comment]
There's a PR here that seems to address this issue - https://github.com/jenv/jenv/pull/244

I'd also like to use `jenv shell 11` instead of `jenv shell 11.0`.

@gcuisinier @doctorpangloss could either of you review #244 please?

**Comment by antonmos** [comment]
That PR has been long merged, but this seems to be not working again
```
 $ jenv add /Library/Java/JavaVirtualMachines/temurin-17.jdk/Contents/Home/
 temurin64-17.0.9 already present, skip installation
 17.0.9 already present, skip installation
 17.0 already present, skip installation
 17.0.9 already present, skip installation
```

**Comment by andrewflbarnes** [comment]
There was a regression whch has since been fixed and re-released. This should be resolved in the latest version of jenv (0.5.7)
